packages/api-v2/src/main.ts

```diff
@@ -20,8 +20,9 @@ async function bootstrap() {
   const app = await NestFactory.create(AppModule, {
     logger: graduateLogger,
     cors: {
-      origin:
-        "https://graduatenu-frontend-v2-git-christina-move-fro-b625a5-sandboxneu.vercel.app",
+      origin: "https://frontend-staging.graduatenu.com",
+      credentials: true,
+      methods: ["GET", "HEAD", "POST", "PUT", "DELETE", "PATCH"],
```

Comment:
Are you sure ``methods`` is needed?

Reply:
Good question! I did some research and I'm pretty sure it is necessary. Essentially not providing this field allows "simple requests" by default which is only GET, HEAD, and POST. 

I know Graduate uses at least DELETE as well and I figured I might as well enable all of them. (Also, enabling all of them now will prevent a potentially really confusing bug later).